### PR TITLE
Import data from an external Redis Database

### LINKED ISSUE #234: Migrating a Redis from a different provider
Une manière de faire est de se connecter à l'instance scalingo et de déclarer le redis comme slave de celui dont vous souhaitez récupérer les données:

scalingo -a app redis-console
> SLAVEOF <host> <port>
> CONFIG SET masterauth <password>
> INFO replication

Lorsque : master_sync_in_progress est à 0, l'instance Redis a récupérer toutes les données, il suffit de désactiver le mode SLAVE 

> SLAVEOF NO ONE

La base Scalingo aura alors les données de la base d'origine.

### PR DESCRIPTION
**Pull request by Soulou**
Fixes #234

### CODE REVIEW

File: _posts/databases/2000-01-01-redis-data-import.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Importing data from an external Redis database
+modified_at: 2018-02-23 00:00:00
+category: databases
+tags: databases redis dump restore migration
+---
+
+This tutorial aims at transfering all the data from a remote Redis database
+(from another provider) to a Redis instance provisioned through the Scalingo
+Redis Addon.
```
Comment (yannski) [inline _posts/databases/2000-01-01-redis-data-import.md]: link to Redis Addon doc page

Comment (Soulou) [reply _posts/databases/2000-01-01-redis-data-import.md]: Thanks

Comment (Soulou) [reply _posts/databases/2000-01-01-redis-data-import.md]: Done

**Comment by yannski** [comment]
LGTM

**Pull request merged**